components/searchable_user_list/searchable_user_list.jsx

```diff
@@ -168,6 +168,19 @@ export default class SearchableUserList extends React.Component {
 
         if (this.props.term || !this.props.users) {
             usersToDisplay = this.props.users;
```

Comment:
You need to `usersToDisplay = [...this.props.users];` otherwise this is going to modify the prop

Reply:
I made the change there. Thanks